offers/README.md

```diff
@@ -221,7 +223,7 @@ fileName:offer.json
 
 ## Create Offer
 
-To create an offer you can use the [Create Offer](https://reference.fidel.uk/v1/reference#create-offer) API endpoint or the dashboard and specify your offer parameters.
+To create an offer you can use the [Create Offer](https://reference.fidel.uk/v1/reference#create-offer) API endpoint or the dashboard and specify your offer parameters. If you have an account in the [CLO dashboard](https://clo.fidel.uk), you may create an offer there as well.
```

Comment:
let's rename CLO dashboard -> Offers dashboard, that's the preferred usage

Reply:
nice point we are trying to transaction to Offers instead of CLO